game_api.txt

```diff
@@ -1115,3 +1115,28 @@ This function registers a shapeless recipe that takes `ingredient`
 and `result` as input and outputs `result`.
 
 The metadata of the input `result` is copied to the output `result`.
+
+
+Log API
+-------
+
+Logs action of the player with a node at a certain position.
+By default only actions of real players are logged.
+Actions of non-players (usually machines) are logged only when
+setting `enable_non_player_action_log` is enabled.
```

Comment:
`log_non_player_actions` is a better name for this IMO.

Reply:
I aggree.  Adapted in https://github.com/minetest/minetest_game/pull/2941/commits/90591c5a49972fac888da4fb2f0ceaf3237b8526